### DISCUSSION TITLE
10月份更新lantern后，所有客户端专业版突然变成免费版，无法收到验证码，请帮助解决

### DISCUSSION THREAD

**Discussion by wx1yyn** [getlantern/lantern-forum-cn#976] [Q&A]
@Mike777Khoo 你好，我看到论坛里好多出现这个问题的人，请帮助解决，注册邮箱和支付订单已发到你[mike@getlantern.org](mailto:mike@getlantern.org)邮箱，如果没收到请回复下，我的蓝灯注册邮箱是2502710162@qq.com，会用这个给你发邮件

**Comment by MaxMKMax** [comment]
@wx1yyn 好好的. 我已经帮您提交问题上去了.

**Reply by wx1yyn** [reply]
@Mike777Khoo 感谢，已经解决了

**Comment by wx1yyn** [comment]
已解决
